index.md

```diff
@@ -23,7 +23,7 @@ Note: We will be reserving 20% of tickets for attendees travelling from outside
 
 ### What happens at SoCraTes?
 
-The gathering runs from **Thursday 7th June** to **Sunday 10th June, 2018**.
+The gathering runs from over 4 days during a week in **June**, attendees arriving on **Thursday** and departing on **Sunday**.
```

Comment:
is "from over" correct? 
I think it should be just "for"

Reply:
Fixed :)